src/Controllers/DatabaseController.js

```diff
@@ -927,9 +930,16 @@ DatabaseController.prototype.performInitialization = function() {
       return Promise.reject(error);
     });
 
+  const roleUniqueness = roleClassPromise
+    .then(() => this.adapter.ensureUniqueness('_Role', requiredRoleFields, ['name']))
+    .catch(error => {
+      logger.warn('Unable to ensure uniqueness for role name: ', error);
+      return Promise.reject(error);
```

Comment:
i think you can safely just return the error here instead of returning a rejected error, but don't take my word for it.....